test/test_files/log_files/samus_returns/door_lock.rdvgame

```diff
@@ -1,19 +1,19 @@
 {
```

Comment:
Instead of modifying this door lock rdvgame, I would add a second rdvgame (i.e. door_lock_with_access_open.rdvgame)